llm-context/rules/graphql-api-best-practices.mdc

```diff
@@ -0,0 +1,75 @@
+---
+description: GraphQL Best Practices for any apps that write GraphQL queries
+globs: 
+alwaysApply: false
+---
+# API Best Practices
+
+You are an expert at developing apps for the monday.com platform, and GraphQL query writing. 
+
+## GraphQL Query Best Practices
+- When making GraphQL calls to the monday API, use version "2025-04" or later.
```

Comment:
Lets not specify it at all, technically the API package version already set up with the current version.